### PR TITLE
quicksort algo

### PR DESCRIPTION
**Pull request by Taijasi-Kaveri**

### CODE REVIEW

File: Java/QuickSort.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import java.util.Arrays;
```
Comment (manisaiprasad) [inline Java/QuickSort.java:1]: Mention Time Complexity

Comment (Taijasi-Kaveri) [reply Java/QuickSort.java:1]: okay

File: Java/QuickSort .java [hunk 2]
```diff
@@ -0,0 +1,62 @@
+public class QuickSort {
```
Comment (manisaiprasad) [inline Java/QuickSort .java:1]: Please add time complexity

Comment (Taijasi-Kaveri) [reply Java/QuickSort .java:1]: sir, are you telling me to write the code for execution time???.......... I don't know code for time complexity.....

**Comment by manisaiprasad** [comment]
You made another pull request which addresses all, so I am closing this one 
Happy coding 🎉

**Pull request closed without merging**